### DISCUSSION TITLE
Does PubPub/KFG deposit reference metadata to Crossref?

### DISCUSSION THREAD

**Discussion by aimeediehl** [knowledgefutures/pubpub#2935] [Support]
Good morning! Can you please tell me if PubPub automatically deposits reference metadata to Crossref? 

I've been investigating the DOAJ application process and noticed that one question asks whether a journal complies with [I4OC](https://i4oc.org/#goals) standards for open citations. Info on that page says:

"Many publishers currently deposit reference lists from their journal articles to Crossref as part of their participation in Crossref’s Cited-by service. To open their references, along with the other bibliographic metadata that publishers send to Crossref, publishers need to turn on reference distribution for all of the Digital Object Identifier (DOI) prefixes they manage. This step allows references within the Crossref members’ articles to be distributed without restriction through all of Crossref's Metadata Delivery services, including the REST API and bulk metadata dumps, to any interested party. See below for additional information on reference distribution and on how to participate in Crossref’s Cited-by service."

Our journal wants to apply to the DOAJ when eligible and is using the application questions as a guide for developing our community and its content.

Thanks for any info you have on this.

**Comment by kyledenlinger** [comment]
I might be wrong, but while this doesn't happen automatically, it's easy to deposit an issue and articles once they're published. [More here.](https://help.pubpub.org/pub/creating-a-journal-on-pubpub#deposit-your-journal)

**Comment by aimeediehl** [comment]
Hi Kyle, 
We are already depositing our pubs to CrossRef via PubPub—we get a DOI as soon as we publish—but I'm unable to tell if the reference lists for each article are deposited. It looks like PubPub might only deposit basic article metadata. Do you have any more specific information about compliance with the ([I4OC](https://i4oc.org/#goals)) standards described in my original request?

**Reply by kyledenlinger** [reply]
Looking at [PubPub's Crossref README](https://github.com/pubpub/pubpub/blob/master/utils/crossref/README.md), it doesn't appear that references are included in the deposit.
